### PR TITLE
fix: change webhook for the upstream nginx version

### PR DESCRIPTION
**Pull request by hurtadosanti**
https://github.com/jet/kube-webhook-certgen/issues/30

### CODE REVIEW

**Review by sbaier1 — APPROVED** [review]
LGTM, i assume it's a drop-in replacement then?

File: charts/hivemq-operator/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# chart 0.9.4
```
Comment (sbaier1) [inline charts/hivemq-operator/CHANGELOG.md:1]: btw, you need to update the Chart.yaml to reflect the version update

**Review by sbaier1 — CHANGES_REQUESTED** [review]
oversaw the missing Chart.yaml version update, please add it

**Comment by hurtadosanti** [comment]
> Yes I test it with Kubernetes using k3s versions {"v1.18.20-k3s1", "v1.19.16-k3s1", "v1.20.15-k3s1", "v1.21.10-k3s1", "v1.22.7-k3s1", "v1.23.4-k3s1"}

**Review by sbaier1 — APPROVED** [review]
LGTM 👍

**Pull request merged**